fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -551,6 +551,19 @@ private String paramsToSql() {
             sb.append("DISTINCT ");
         }
         int len = children.size();
+
+        if (fnName.getFunction().equalsIgnoreCase("char")) {
+            for (int i = 1; i < len; ++i) {
+                sb.append(children.get(i).toSql());
+                if (i < len - 1) {
+                    sb.append(", ");
+                }
+            }
+            sb.append(" using ");
+            sb.append(children.get(0).toSql().replace("'", "")).append(")");
```

Comment:
i think we should use substr() to remove the first and the last character. replace will generate wrong sql, think about `'ab\'c'`